webapp-django/crashstats/crashstats/views.py

```diff
@@ -1050,14 +1049,23 @@ def crashes_per_user(request, default_context=None):
     default = request.session.get('date_range_type', 'report')
     context['date_range_type'] = params.get('date_range_type') or default
 
-    api = SuperSearchUnredacted()
-    # XXX HACK! Because of a bug in SuperSearch we can't search for
-    # '4X.0b' so we have to change that to '4X.0b1' :(
-    # See https://bugzilla.mozilla.org/show_bug.cgi?id=1197251
+    params['original_versions'] = params['versions'][:]  # make a copy
```

Comment:
I'm not convinced. Neither did I read that long blog post. 

You can do `[:]` on a tuple too and what you get is a copy. It's the equivalent of `import copy; thing = copy.copy(otherthing)`. If you do `list(something)` you always get a list. Even if it was a tuple. Also, if you do `[:]` you **know** it must be a list (or tuple). It can't be a generator because generators don't support slicing. 

I see `list()` often being used in cased where you desperately need to exhaust a generator. I.e. asking it to return all its elements right here right now. 

If you were to use `list()` instead of `[:]` you'd still need to have the... ` # make a copy` comment because it tells the reader that that's why you're doing it. Not because it was a generator that needs to be fully iterated. 

In fact, with `[:]` there is no need to say `# make a copy` because, of course, it can only be a list. 

I guess I'm so used to `[:]` that I was blind to your gut reaction that `list()` "[is] easier to understand". I can change it if you like.